src/main/java/com/google/sps/servlets/LocationServlet.java

```diff
@@ -96,6 +97,9 @@ private void jsonToMap(String jsonString) {
             case "route" :
               geoMap.put("Street Name", name);
               break;
+            case "country" :
+              geoMap.put("Country", shortName);
+              break;
```

Comment:
toogle.js is using full country name in line 119, so if we're going to be using the short name now it would be useful to change that too.